website/docs/r/project.html.markdown

```diff
@@ -58,6 +63,22 @@ Teams attribute is optional
 
 ~> **NOTE:** Project created by API Keys must belong to an existing organization.
 
+### Api Keys
+api_keys attribute is optional
+
+~> **NOTE:** Atlas limits the number of api keys to a maximum of 500 api keys per organization.
```

Comment:
```suggestion
```
Since we can't create a key here, just assign, the limitation information doesn't match the context.